cfg/Parameters.cfg

```diff
@@ -40,9 +40,9 @@ PACKAGE='control_toolbox'
 
 def generate(gen):
 	#        Name            Type    Level  Description      Default  Min  Max
-	gen.add( "p" ,      double_t, 1,"Proportional gain.", 10.0 , 0 , 100000)
-	gen.add( "i" ,      double_t, 1,"Integral gain.",     0.1 , 0 , 1000)
-	gen.add( "d" ,      double_t, 1,"Derivative gain.",   1.0 , 0 , 1000)
+	gen.add( "p" ,      double_t, 1,"Proportional gain.", 10.0 , -100000 , 100000)
```

Comment:
hey @efernandez. I can think of the following system: G(s) = -1/s-3 which is unstable.
Adding a P controller results in transfer function of -p/(s-3-p) and this becomes stable for p < -3. Also if you want to add an integral part it should be negative. Generally p and I should have same sign.